.gitignore

```diff
@@ -110,3 +110,6 @@ src/scripts/mondo_unmapped.tsv
 # Test
 tests/output/
 src/scripts/dataframes/*
+src/ontology/reports/gard.subclass.added-obsolete.robot.tsv
+src/ontology/reports/gard.subclass.added.robot.tsv
+src/ontology/reports/gard.subclass.confirmed.robot.tsv
```

Comment:
Removed the gard ones as they are meaningless for this process.

Reply:
Sounds good. I have it high on my todo list to do teardown of GARD.